src/zh/rumanhua/src/eu/kanade/tachiyomi/extension/zh/rumanhua/PageDecrypt.kt

```diff
@@ -0,0 +1,111 @@
+package eu.kanade.tachiyomi.extension.zh.rumanhua
+
+import android.util.Base64
+import org.jsoup.nodes.Document
+
+// all2.js?v=2.3
+class PageDecrypt {
+    private val SCRIPT_PATTERN = "eval(function(p,a,c,k,e,d)"
+    private val CONTENT_MARKER = "var __c0rst96=\""
+
+    fun toDecrypt(doc: Document): String {
```

Comment:
Consider using `document` per convention:
```suggestion
    fun toDecrypt(document: Document): String {
```
(needs other changes)